chalice/awsclient.py

```diff
@@ -104,10 +109,16 @@ def delete_function(self, function_name):
         except lambda_client.exceptions.ResourceNotFoundException:
             raise ResourceDoesNotExistError(function_name)
 
-    def update_function(self, function_name, zip_contents,
-                        environment_variables=None,
-                        runtime=None, tags=None):
-        # type: (str, str, _STR_MAP, _OPT_STR, _STR_MAP) -> Dict[str, Any]
+    def update_function(self,
+                        function_name,  # type: str
```

Comment:
Please align the type definitions vertically.  There's an example of this in [pep 484](https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code).